### PR TITLE
SW-3268 Visual Bugfixes

### PR DESCRIPTION
**Pull request by charlie-tf**
- fix margin around report content frame
- fix mobile button layout (stacked instead of horizontal)

image
image

### CODE REVIEW

**Comment by charlie-tf** [comment]
@chudo76 I commented on this in the jira, but please let me know if you want 32px around the content frame. Thanks!

File: src/components/Reports/ReportForm.tsx [hunk 1]
```diff
@@ -311,5 +313,6 @@ export default function ReportForm(props: ReportFormProps): JSX.Element {
         )}
       </Grid>
     </Grid>
+    // </Box>
```
Comment (karthikbtf) [inline src/components/Reports/ReportForm.tsx:316]: remove the comments here and above?

File: src/components/common/FormBottomBar.tsx [hunk 2]
```diff
@@ -90,7 +94,7 @@ export default function FormBottomBar({
         type='passive'
         className={classes.button}
       />
-      <div>
+      <div className={classes.rightButtonGroup}>
```
Comment (karthikbtf) [inline src/components/common/FormBottomBar.tsx:97]: thanks for fixing.. i see it broken in staging for other pages with a form bottom bar.
curious.. when/how did this break?

**Review by karthikbtf — APPROVED** [review]
👍

File: src/components/Reports/ReportForm.tsx (hunk 1 above)
Comment (charlie-tf) [reply src/components/Reports/ReportForm.tsx:316]: Oops, thanks

File: src/components/common/FormBottomBar.tsx (hunk 2 above)
Comment (charlie-tf) [reply src/components/common/FormBottomBar.tsx:97]: It's likely i broke it when i was adding the ability to display more buttons in the form

Comment (karthikbtf) [reply src/components/common/FormBottomBar.tsx:97]: all cool, glad you caught it, thanks much 🙏

**Pull request merged**